crates/typst/src/model/numbering.rs

```diff
@@ -297,6 +297,10 @@ pub enum NumberingKind {
     EasternArabic,
     /// The variant of Eastern Arabic numerals used in Persian and Urdu.
     EasternArabicPersian,
+    /// Devanagari numerals.
+    // TODO: Is it the best name? (I use the suffic ‘Number’ in case we want to also support
+    // devanagari letters)
```

Comment:
Name seems fine. It's also not that important. What matters is the `from_char` and that makes sense.